kiota_http/httpx_request_adapter.py

```diff
@@ -347,3 +347,7 @@ def get_request_from_request_information(
         )
         request.options = request_info.request_options  # type:ignore
         return request
+
+    async def convert_to_native_async(self, request_info: RequestInformation) -> httpx.Request:
+        request = self.get_request_from_request_information(request_info)
```

Comment:
Need to authenticate the request first before conversion

Reply:
@Ndiritu thanks! Should be good now.